calico-vpp-agent/Makefile

```diff
@@ -11,20 +11,23 @@ all: build gobgp image
 
 export GOOS=linux
 
+bin:
+	mkdir -p bin
+
 # We make felix-api-proxy a static executable as it will run in the calico container
 # for which we have less control on the env and glibc version
 .PHONY: felix-api-proxy
 felix-api-proxy: CGO_ENABLED=0
-felix-api-proxy:
-	${DOCKER_RUN} go build -o ./cmd/felix-api-proxy ./cmd/api-proxy
+felix-api-proxy: bin
+	${DOCKER_RUN} go build -o ./bin/felix-api-proxy ./cmd/api-proxy
 
-build: felix-api-proxy
-	${DOCKER_RUN} go build -o ./cmd/calico-vpp-agent ./cmd
-	${DOCKER_RUN} go build -o ./cmd/debug ./cmd/debug-state
+build: felix-api-proxy bin
```

Comment:
`felix-api-proxy` target already calls `bin`, I think this "bin" can be deleted

Reply:
Agreed, just left it there in case at some point we decide to remove `felix-api-proxy`